### PR TITLE
Update GERP files for e106

### PR DESCRIPTION
**Pull request by helensch**

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json [hunk 1]
```diff
@@ -840,143 +840,143 @@
...
       "assembly": "TETRAODON8",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-105/compara/conservation_scores/65_fish.gerp_conservation_score/gerp_conservation_scores.tetraodon_nigroviridis.TETRAODON8.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-106/compara/conservation_scores/65_fish.gerp_conservation_score/gerp_conservation_scores.tetraodon_nigroviridis.TETRAODON8.bw",
       "annotation_type": "gerp"
     },
     {
       "id": "65_fish.gerp_conservation_score",
       "species": "salmo_salar",
       "assembly": "ICSASG_v2",
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json:977]: The assembly also needs to be updated here.

Comment (dglemos) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json:977]: I meant the salmon assembly from ICSASG_v2 to Ssal_v3.1.

Comment (helensch) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: Thanks I have updated the assembly for the GERP file.
I will do a separate PR for the removal of the Salmon VCF for the e106 release.

Comment (dglemos) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: Thanks!

**Review by dglemos — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/106 and master

**Pull request merged**